plugins/modules/aci_l3out_logical_node.py

```diff
@@ -295,12 +306,27 @@ def main():
             module_object=tdn,
             target_filter={"name": tdn},
         ),
+        child_classes=child_classes,
     )
 
     aci.get_existing()
 
+    if loopback_address is not None:
+        if loopback_address == "" and isinstance(aci.existing, list) and len(aci.existing) > 0:
+            for child in aci.existing[0].get("l3extRsNodeL3OutAtt", {}).get("children", []):
+                existing_loopback_interface_profile = child.get("l3extLoopBackIfP")
+                if existing_loopback_interface_profile:
+                    previous_loopback_address = existing_loopback_interface_profile.get("attributes",{}).get("addr")
+                    child_configs.extend([dict(l3extLoopBackIfP=dict(attributes=dict(addr=previous_loopback_address,status="deleted")))])
```

Comment:
can this be combined?
```suggestion
                loopback_addres = child.get("l3extLoopBackIfP", {}).get("attributes",{}).get("addr")
                if loopback_addres:
               child_configs.extend([dict(l3extLoopBackIfP=dict(attributes=dict(addr=loopback_addres, status="deleted")))])
```